WebView/sampleData/weather.json

```diff
@@ -0,0 +1,29 @@
+{
+  "newYork":{
+    "currentTemp":"75°",
+    "highTemp":"83°",
+    "lowTemp":"64°",
+    "chancePrecip":"90%",
+    "humidity":"23%",
```

Comment:
Can we change each of these to Numbers and let the app add the appropriate units? Also, temperature should clarify which units (C or F).

Reply:
I will make these changes to the JSON here and then I will create a follow up PR changing where the app is fetching data from along with the adjustments needed on the app side